payment_bitcoin/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===============
+Payment Bitcoin
+===============
+
+Bitcoin Transfer Payment Acquirer
```

Comment:
Move this sentence as title ('Bitcoin Transfer Payment Acquirer') and replace with the one at the end of paragraph